src/Microsoft.Diagnostics.Monitoring.StartupHook/Exceptions/Identification/ExceptionGroupIdentifierCache.cs

```diff
@@ -83,13 +83,19 @@ public ulong GetOrAdd(MethodBase method)
                 return methodId;
 
             // Dynamic methods do not have metadata tokens
-            uint metadataToken = 0;
+            uint methodToken = 0;
             try
             {
-                metadataToken = Convert.ToUInt32(method.MetadataToken);
+                methodToken = Convert.ToUInt32(method.MetadataToken);
```

Comment:
Double checking -- this appears to be a bug fix, correct? We were incorrectly reporting the method token as the parent class token previously?

Reply:
Correct